src/index.css

```
@@ -234,6 +234,7 @@
   }
   body {
     @apply bg-background text-foreground;
+    font-family: var(--font-sans);
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)